bool dir
int8 tSpeed
int8 fwdSpeed
bool r_proxy_flag
bool l_proxy_flag

bool r_corner
bool l_corner